.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -65,6 +65,7 @@ Branch name: `bugfix_<Issue Number>_main_<Version>_<Description>`
 - [ ] Add/update log messages for easier debugging.
 - [ ] Add/update unit tests.
 - [ ] Add/update documentation.
+- [ ] Add any new Python packages to the [METplus Components Python Requirements](https://metplus.readthedocs.io/en/develop/Users_Guide/overview.html#metplus-components-python-requirements) table.
```

Comment:
I think it is probably unlikely that a bugfix would warrant updating the table, but it doesn't hurt to have it in here.